tests/TestCase/View/Widget/DateTimeWidgetTest.php

```diff
@@ -861,14 +862,14 @@ public function testRenderMinuteWidget()
         ], $this->context);
         $this->assertContains('<select name="date[minute]" data-foo="test">', $result);
         $this->assertContains(
-            '<option value="00">00</option>',
+            '<option value="00">0</option>',
             $result,
-            'contains 00'
+            'contains 0'
         );
         $this->assertContains(
-            '<option value="05">05</option>',
+            '<option value="05">5</option>',
```

Comment:
This could be interpreted as a regression by some people. Can we keep the format the same?